specs/regional_communities.md

```diff
@@ -1,61 +1,333 @@
 
-### Actions
+A merge proposal should be approved by majority vote in both communities before it is allowed to happen.
 
-List the actions each collective will take individually.
+It's not easy to design rules for how the merge should be executed, based on the complexity of possible permutations of the pre-merge Communities. It seems plausible that merge power should not be given to regular users but could be given as an admin power to a dev / superuser. The alternative would be that merging not be supported, in which case the community will manually have to move from and abandon one community in favor of the other.
 
-Describe which action is done by which actors. Feel free to use a table format or provide your own graphics or bundle the description in the Actions section. A "swimlane process chart" often works well here.
+### Interact with my Community
+Members should at minimum be able to post messages within the community which become publicly readable to anyone regardless of membership status, and where other members are allowed to comment on the message.
 
-|         | Action 1 | Action 2 | Action 3 | action 4 |
+Messages posted on boards that I follow should show up in my personal feed in chronological order. But they don't show up in the feed of users who are not members and don't follow the community.
+
+Members or admins (depending on Community settings) should be able to post polls for the community, which only community members are allowed to vote in.
+
+As a member I should be able to see all other members listed in a sidebar, along with their membership / admin status.
+
+### Ban a user from a Community
+Banning is a strong power and should be used very restrictively. We suggest milder and more human forms of restricting users that are not following the code of conduct for a community. 
+- Issue a warning for breach of code of conduct
+- Temporarily silence the user for a day
+- Temporarily silence the user for a week
+- Silence the user until un-silenced (perhaps after taking some corrective action)
+- Banning the user with chance of reapplying
+- Banning the user with chance of reapplying (permanently)
+
+It is this author's opinion that silencing the user is a much preferred method over banning them. Silencing allows the user to keep following the activitiy of the community and still feel like they belong there. It creates less animosity and makes the world a nicer place. Consider not implementing the banning feature.
+
+These measures can be designed such that each increasing level of banning must be preceeded by the milder forms before it can be implemented. E.g. must silence the user for a day before can silence them for a week.
+
+### Manage a Community
+Communities should be open and welcoming, not closed for elite members only. We should design them such that members have as much powers as possible, and that admins are given as few god-like options as possible.
```

Comment:
idea: no admin by default. Community  is DAO

Reply:
No admin: I like it. But how do we prevent potentially disastrous actions, such as a malicious member renaming the community and changing the purpose? A majority community vote is not a good option, as it's proven to be very hard to get quorum votes within loosely affiliated and large communities.

Reply:
Community is DAO: Needs definition of what a DAO is. If the admins are thought of as Council members then that's a DAO too.

Reply:
> such as a malicious member renaming the community and changing the purpose? A majority community vote is not a good option, as it's proven to be very hard to get quorum votes within loosely affiliated and large communities.

Each community should have it's parameters to reach consensus (eg, min quorum, could be % or fixed number, eg 10)